packages/experimental/NativeFontMetrics/src/NativeFontMetrics.tsx

```diff
@@ -0,0 +1,24 @@
+import { NativeModules } from 'react-native';
+
+export const NativeFontMetrics = NativeModules.FRNFontMetrics;
+
+export type TextStyle =
+  | 'caption2'
+  | 'caption1'
+  | 'footnote'
+  | 'subheadline'
+  | 'callout'
+  | 'body'
+  | 'headline'
+  | 'title3'
+  | 'title2'
+  | 'title1'
+  | 'largeTitle';
+
+export type ScaleFactors = { [K in TextStyle]: number | undefined };
+
+interface NativeFontMetricsInterface {
```

Comment:
Out of curiosity which platforms is this meant to run on, all of them? Just iOS? Should the package be stubbed for platforms that won't provide useful values?

Reply:
Yeah, it's only meant to be run on iOS. We can stub this out for other platforms.

Reply:
Resolved with a4f879f.